pandas/tests/extension/test_string.py

```diff
@@ -85,6 +85,14 @@ class TestMethods(base.BaseMethodsTests):
     def test_value_counts(self, all_data, dropna):
         return super().test_value_counts(all_data, dropna)
 
+    def test_memory_usage(self, data):
```

Comment:
Can you move this test to `/tests/arrays/string_/test_string.py` ? 

(the tests in /extension/ are all common tests (shared among dtypes), while in /arrays/ we keep specific tests)